docs/assemblies/proc_deploying-in-disconnected-environments.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_deploying-in-disconnected-environments"]
+= Deploying OpenStack in a disconnected environment
+
+[role="_abstract"]
+== Process
+Deploying in disconnected environments can be achieved largely by following the OpenShift documentation for mirroring OLM Operators: https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images
+
+== Technical Implementation
+The details provided in this section are for informational purposes only. Users should not need to interact with anything additional after completing the above mentioned OLM mirroring process.
+
+The `openstack-operator` contains a list of related images that will ensure all required images for the deployment are mirrored following the above OpenShift process. Once images are mirrored, the `ImageContentSourcePolicy` custom resource (CR) is created. This process results in a `MachineConfig` called `99-master-genereted-registries` being updated in the cluster. The `99-master-generated-registries` `MachineConfig` contains a `registries.conf` file that is applied to all of the OpenShift nodes in the cluster.
+
+In order to integrate cleanly with this process. The dataplane controller will check for the existence of an `ImageContentSourcePolicy`. If one is found, we will read the `registries.conf` file from the `99-master-generated-registries` `MachineConfig`. The dataplane controller will then set two variables in the Ansible inventory for the nodes.
```

Comment:
s/dataplane controller/openstack-operator ?

The first sentence is also a fragment:
`In order to integrate cleanly with this process.`

I would rewrite this as:
```
In order for EDPM to integrate cleanly with this process, openstack-operator checks for the existence of an `ImageContentSourcePolicy`.
```

Reply:
I removed all of the "we"'s as well. I just swapped EDPM for dataplane since I hadn't already referenced the acronym anywhere else, I just stuck with dataplane.